packages/turbo-codemod/src/commands/migrate/steps/getTurboUpgradeCommand.ts

```diff
@@ -56,8 +56,8 @@ function getLocalUpgradeCommand({
           `turbo@${to}`,
           installType === "devDependencies" && "--dev",
         ]);
-        // yarn 1.x
```

Comment:
This confused me for a longer moment than I'd care to admit. I moved it down below the closing brace since it makes more sense there.